showcase/app/templates/components/tag.hbs

```diff
@@ -33,12 +33,22 @@
   </Shw::Flex>
 
   <Shw::Flex @label="With long text" as |SF|>
-    <SF.Item {{style width="200px"}}>
+    <SF.Item {{style width="176px"}}>
```

Comment:
Suggestion: These examples don't need the `width` set now that there is a `max-width` on the tag text.

<img width="1005" alt="Screenshot 2025-02-11 at 9 48 39 AM" src="https://github.com/user-attachments/assets/2f1bacbe-b35f-4e77-b9a6-487feb1c9992" />